### PR TITLE
Update about us ug dg

### LINKED ISSUE #17: Update the UG (Help command)

### LINKED ISSUE #19: Update the DG : Target user profile, value proposition, and user stories

### PR DESCRIPTION
**Pull request by jannaleong**
Fixes #17 
Fixes #19

### CODE REVIEW

**Review by chiageng — APPROVED** [review]
LGTM

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -262,27 +262,26 @@ _{Explain here how the data archiving feature will be implemented}_
...
-* has a need to manage a significant number of contacts
-* prefer desktop apps over other types
-* can type fast
-* prefers typing to mouse interactions
-* is reasonably comfortable using CLI apps
+* Dog cafe owners who need to manage a team of staff, F&B vendors & a dog maintainence team.
+* Prefer typing over other types and is comfortable using CLI applications.
 
-**Value proposition**: manage contacts faster than a typical mouse/GUI driven app
+**Value proposition**: PoochPlanner is a desktop application to track details of various groups (vendors, staff, dog maintainence) that dog cafe owners have to regularly interact with. The app is optimised for use using
```
Comment (yleeyilin) [inline docs/DeveloperGuide.md:268]: You may want to split the line for value proposition here.

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -262,27 +262,26 @@ _{Explain here how the data archiving feature will be implemented}_
...
-| `* * *`  | user                                       | find a person by name          | locate details of persons without having to go through the entire list |
-| `* *`    | user                                       | hide private contact details   | minimize chance of someone else seeing them by accident                |
-| `*`      | user with many persons in the address book | sort persons by name           | locate a person easily                                                 |
+| Priority      | <div style="width:50px">As a …​</div> | I want to …​                                               | So that I can…​                                             |
+|---------------|---------------------------------------|------------------------------------------------------------|-------------------------------------------------------------|
+| `* * *`       | well connected user                   | search contacts                                            | I can save time                                             |
```
Comment (yleeyilin) [inline docs/DeveloperGuide.md:278]: Can you standardise part 3 for the use case.

**Review by yleeyilin — COMMENTED** [review]
Overall looks good, just left some minor comments.

**Review by jamessinmaojun — APPROVED** [review]
lgtm

**Comment by chiageng** [comment]
@jannaleong @yleeyilin @jamessinmaojun @Joshy837 
There is an update in team repo, please pull into your local fork

**Pull request merged**